lib/Server.js

```diff
@@ -174,7 +174,14 @@ class Server {
 
     // ref: https://github.com/webpack/webpack-dev-server/issues/1575
     // remove this when send@^0.16.3
-    express.static.mime.types.wasm = 'application/wasm';
+    if (
+      express.static &&
+      express.static.mime &&
+      express.static.mime.types &&
+      !express.static.mime.types.wasm
```

Comment:
I think better to remove `!express.static.mime.types.wasm` because in some old cases it can be `text/html` and it is invalid, we need use right mime for `wasm`